File: Java/IntroJava/02.Operators and Expressions/src/PointInCircleOutOfRectangle.java
/*
 * Write an expression that checks for given point (x, y) 
 * if it is within the circle K( (1,1), 3) and out of 
 * the rectangle R(top=1, left=-1, width=6, height=2).
*/
import java.util.Scanner;
public class PointInCircleOutOfRectangle {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int x = input.nextInt();
        int y = input.nextInt();
        input.close();
        boolean boolInRectangle = ((x >= -1) && (x <= 5)) && ((y <= 1) && (y >= -1));
        boolean boolInCircle = Math.sqrt((x - 1) * (x - 1) + (y - 1) * (y - 1)) <= 3;
        boolean boolResult = (boolInRectangle == false) && (boolInCircle == true);
        System.out.printf("The point in the area /cirle - rectangle/ = %s %n", boolResult);
	}

}

File: CSharp/Unit1/02.Operators and Expressions/09.PointInCircleOutOfRectangle/PointInCircleOutOfRectangle.cs
/*
 * Write an expression that checks for given point (x, y) 
 * if it is within the circle K( (1,1), 3) and out of 
 * the rectangle R(top=1, left=-1, width=6, height=2).
*/
using System;
class PointInCircleOutOfRectangle
{
    static void Main()
    {
        int x = int.Parse(Console.ReadLine());
        int y = int.Parse(Console.ReadLine());
        bool boolInRectangle = ((x >= -1) && (x <= 5)) && ((y <= 1) && (y >= -1));
        bool boolInCircle = Math.Sqrt((x - 1) * (x - 1) + (y - 1) * (y - 1)) <= 3;
        bool boolResult = (boolInRectangle == false) && (boolInCircle == true);
        Console.WriteLine("The point in the area /cirle - rectangle/ = {0}", boolResult);
    }
}
